commit 44529f8975b7b93709b1b92be7d027a1d406de8a
Author: Chao Yu <chao@kernel.org>
Date:   Mon Jun 12 09:44:24 2017 +0800

    f2fs: fix to show injection rate in ->show_options
    
    If fault injection functionality is enabled, show additional injection
    rate in ->show_options.
    
    Signed-off-by: Chao Yu <yuchao0@huawei.com>
    Signed-off-by: Jaegeuk Kim <jaegeuk@kernel.org>

diff --git a/fs/f2fs/super.c b/fs/f2fs/super.c
index fe86a7edfa60..2aa864eae522 100644
--- a/fs/f2fs/super.c
+++ b/fs/f2fs/super.c
@@ -984,7 +984,8 @@ static int f2fs_show_options(struct seq_file *seq, struct dentry *root)
 		seq_printf(seq, ",io_size=%uKB", F2FS_IO_SIZE_KB(sbi));
 #ifdef CONFIG_F2FS_FAULT_INJECTION
 	if (test_opt(sbi, FAULT_INJECTION))
-		seq_puts(seq, ",fault_injection");
+		seq_printf(seq, ",fault_injection=%u",
+				sbi->fault_info.inject_rate);
 #endif
 
 	return 0;